salt/_states/metalk8s_kubernetes.py

```diff
@@ -472,6 +472,58 @@ def namespace_present(name, **kwargs):
     return ret
 
 
+def namespace_annotation_present(
+        name,
+        annotation_key,
+        annotation_value,
+        **kwargs):
+    '''
+    Ensures that the namespace is patched.
+
+    name
+        The name of the namespace
+
+    annotation_key
+        Key of the annotation
+    annotation_value
+        Value of the annotation
+    '''
+
+    ret = {'name': name,
+           'changes': {},
+           'result': False,
+           'comment': ''}
+
+    annotation = __salt__['metalk8s_kubernetes.show_namespace'](
```

Comment:
This function give you all the namespace content not only the annotation so that cannot work

Reply:
Well, my thinking here is that `show_namespace` returns a dictionary. I can be more specific by drilling down to `annotations` within the dict

Reply:
Yes you can do that also but as you created a function to only patch annotation you should also create one to only list them